frontend/src/modules/app/OverlayScreen.tsx

```diff
@@ -77,8 +77,10 @@ const AudioAlarm = (): JSX.Element => {
   const alarmMuteStatus = useSelector(getAlarmMuteStatus, shallowEqual);
   const [audio] = useState(new Audio(`${process.env.PUBLIC_URL}/alarm.mp3`));
   audio.loop = true;
+  const storeData = store.getState();
+  const events = storeData.controller.eventLog.nextLogEvents.elements;
   const [playing, setPlaying] = useState(alarmMuteStatus.active);
-
+  const patientLogEvent = events.find((el: LogEvent) => (el.type as number) < 2);
```

Comment:
This function looks like it was inspired by the `getPopupEventLog` selector. Maybe we should make this be its own selector?